app/src/main/java/com/example/travellens/fragments/PostsFragment.java

```diff
@@ -91,6 +97,7 @@ public class PostsFragment extends Fragment {
     private static final String TAG = "POSTS_FRAGMENT";
     private final static String KEY_LOCATION = "location";
     private AutocompleteSupportFragment autocompleteFragment;
+    private List<String> typesToFilterBy = new ArrayList<>();
```

Comment:
For better code readability, it's better to group all static variables together and non-static fields together.